### DISCUSSION TITLE
Use the generic client profile with HTTPS

### DISCUSSION THREAD

**Discussion by jpop57-akk** [PTCInc/Universal-Device-Driver-Examples#33] [Q&A] [answered]
Hi everyone,
I'm not very familiar with .js, but i'd like to know if it's hard to make it work with HTTPS. I've try changed port 80 to 443, in vain...
If there is some few lines to change/add, can you tell me please ? Thanks

**Comment by rlabbeptc** [comment, marked as answer]
Hi @jpop57-akk ,

Unfortunately, any encrypted connections with the Universal Device Driver are not supported at this time. You can find other information in this item here: [Issue 30](https://github.com/PTCInc/Universal-Device-Driver-Examples/issues/30)

**Reply by jpop57-akk** [reply]
Thanks a lot for your answer. I'll check that :)
